### ISSUE TITLE
How to use the pretrained five models to test my own data?

### ISSUE THREAD

**Issue by EthanGreen75** [wtomin/MIMA-Net#2]
In main.py of Aff_wild_exps, Even in the test procedure, the train data is still needed. Should I just modify the main.py, leave alone the training part to perform the test procedure?

**Comment by wtomin** [comment]
In the testing procedure, the training data is used to provide the mean and standard deviation of training set labels. This is because we find that using the mean and standard deviation of training set labels to "correct" our model's prediction can make it  perform better on the test set. But this improvement is minor. If you remove it, I think our model can still give you reasonable results.
